modules/integration/src/test/scala/scala/cli/integration/ExportTestProjects.scala

```diff
@@ -270,4 +270,12 @@ object ExportTestProjects {
          |""".stripMargin
     TestInputs(os.rel / "Test.scala" -> testFile)
   }
+
+  def logbackBugCase(scalaVersion: String): TestInputs =
+    TestInputs(os.rel / "script.sc" ->
+      s"""//> using scala "$scalaVersion"
+         |//> using lib "ch.qos.logback:logback-classic:1.4.5"
+         |///> using testFramework ""
```

Comment:
Remove the commented `using` directive `testFramework`

Reply:
fixed